crates/dslab-mp-python/python/dslabmp.py

```diff
@@ -97,12 +99,20 @@ def __init__(self, t: JSON):
         self.inner = t
 
     def serialize(self):
-        return json.dumps(self.inner)
+        def assertion_serializable(o):
+            assert isinstance(o, Message), 'cannot serialize custom classes' 
+            return o.toJSON()
+        return json.dumps(self.inner, default=assertion_serializable)
 
     @staticmethod
-    def deserialize(state):
-        return StateMember(json.loads(state))
+    def obj_hook(msg):
```

Comment:
Предлагаю переименовать функцию и аргумент (там не обязательно сообщение, а просто словарь)

```suggestion
    def _deserialize_message(dct):
```

Reply:
done